packages/manager/apps/web/client/app/hosting/translations/Messages_fr_FR.json

```diff
@@ -1111,7 +1111,7 @@
   "hosting_tab_BOOST_request_title": "Vous allez booster votre offre",
   "hosting_tab_BOOST_request_heading": "Veuillez sélectionner l'offre",
   "hosting_tab_BOOST_request_description": "Vous souhaitez monter sur une offre supérieure, en mode BOOST.",
-  "hosting_tab_BOOST_request_description_billing": "Vous serez facturé au mois plein (du 1 au 31, sans prorata) du montant de l'extra indiqué, une fois par mois, au tarif de l'offre maximale utilisée (si changements multiples).",
+  "hosting_tab_BOOST_request_description_billing": "Vous serez facturé au mois plein (du 1 au 31, sans prorata) du montant de l'extra indiqué, une fois par mois, au tarif de l'offre maximale utilisée (si changements multiples). À partir du 1er novembre 2019, la tarification change et vous serez facturé par jour d'utilisation : 0,49€ / jour et par palier d'offre.",
```

Comment:
Hello @VincentCasse!

Actually I'm a bit out of the context but it seems that this message must be updated at some point and also the price is actually hardcoded which is not an ideal solution.

Could the price be returned by an API?

In order to be able to merge your Pull Request I have some remarks:

- Could you please target the `develop` branch?

_We will update our [CONTRIBUTING.md](https://github.com/ovh/manager/blob/master/CONTRIBUTING.md) to display this information._

- Could you please update your commit message to stick with our [Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)?

**Example**: 
```txt
fix(hosting): change boost pricing description

Announce a price change.
```
Thanks again for your contributions.